src/main.rs

```diff
@@ -44,6 +44,12 @@ struct Args {
         help = "The program to run for each test file"
     )]
     exit_status_prefix: String,
+
+    #[clap(
+        long,
+        help = "Accept what the current output, update the files to match this"
```

Comment:
Grammar is a bit off here, lets change this to `"Update the expected output of each test file to match the actual output"`

Reply:
A much better formulation. Thank you! :D